### ISSUE TITLE
Please included tests in pypi release

### ISSUE THREAD

**Issue by jlec** [xolox/python-capturer#3]
Downstream maintainers love to test, so please include the test suite in the pypi release.

**Comment by xolox** [comment]
Hi Justin and thanks for the feedback! This was kind of already possible (the `capturer.tests` module was already included in PyPI releases) but I did just make it a bit easier when I released version 2.1.1:

``` sh
virtualenv issue-3
pip install 'capturer >= 2.1.1'
python -m capturer.tests
```

Alternatively you can run `python setup.py test` which basically does the same thing in this case. I usually run my Python test suites under pytest and tox but I never require either, so the test suite runs fine without either installed.

**Comment by jlec** [comment]
Thanks for the fix.
